stdlib/ssl.pyi

```diff
@@ -80,6 +80,7 @@ class SSLCertVerificationError(SSLError, ValueError):
 CertificateError = SSLCertVerificationError
 
 if sys.version_info < (3, 12):
+    @deprecated("Deprecated since Python 3.7; Removed in Python 3.12.")
```

Comment:
Is there anything useful to say about what users should use instead?

Also, there should be no capital letter after a semicolon:
```suggestion
    @deprecated("Deprecated since Python 3.7; removed in Python 3.12.")
```